### ISSUE TITLE
Simply nothing happens

### ISSUE THREAD

**Issue by GitSuggestions** [cbdevnet/ipmagnet#17]
Hello, I'm trying to use the tool as the website describes to, but no address is listed when I add the magnet link while connected to the VPN, only when it's not. Is that what is supposted to happen? Thanks.

**Comment by cbdevnet** [comment]
Hi, that may happen if the torrent client is not able to reach the ipMagnet tracker while on the VPN - may I ask which torrent client you are using?
Most clients offer a way to see the tracker responses, ipMagnet returns a message that also shows the detected addresses there - do you see any error messages in that section?

**Comment by GitSuggestions** [comment]
Sorry for taking so much to answer.
qBittorrent, Deluge, Transmission and JSTorrent were the clients I used in the tests. Only JSTorrent failed, I was trying to use VPN browser extensions in combination with JSTorrent. I suppose it's an limitation of JSTorrent and has nothing to do with ipmagnet, yet I was able to download 2 of 3 torrents I was using for the tests. In the other clients I was able download all of them.
